test/services/cache/ExportsCacheModelBuilder.scala

```diff
@@ -117,6 +117,20 @@ trait ExportsCacheModelBuilder {
         BorderTransport(borderModeOfTransportCode, meansOfTransportOnDepartureType, meansOfTransportOnDepartureIDNumber)
       )
     )
+  def withConsignmentReferences(consignmentReferences: Option[ConsignmentReferences]): CacheModifier =
```

Comment:
There's already a `withConsignmentReferences` in this class

Reply:
But in order to use it I need to extra the values out of consigmentRefrences... I just want to pass one in.